proto/autoops/service.proto

```diff
@@ -220,6 +221,78 @@ message ExecuteProgressiveRolloutRequest {
 
 message ExecuteProgressiveRolloutResponse {}
 
+message CreateFlagTriggerRequest {
+  string environment_namespace = 1;
+  CreateFlagTriggerCommand create_flag_trigger_command = 2;
+}
+
+message CreateFlagTriggerResponse {
+  FlagTrigger flag_trigger = 1;
+  string url = 2;
+}
+
+message DeleteFlagTriggerRequest {
+  string environment_namespace = 1;
+  DeleteFlagTriggerCommand delete_flag_trigger_command = 2;
+}
+
+message DeleteFlagTriggerResponse {}
+
+message ControlFlagTriggerRequest {
```

Comment:
Please rename this to `UpdateFlagTriggerRequest` and add the following command.

- ChangeFlagTriggerDescriptionCommand 

Also, to keep consistency with the other APIs, please add an API for enabling and disabling the trigger.

E.g.
https://github.com/bucketeer-io/bucketeer/blob/main/proto/feature/service.proto#L98-L114